libs/public/feature/dashboard/src/lib/components/confirmation/confirmation.component.ts

```diff
@@ -13,11 +14,15 @@ import { NuverialBreadcrumbComponent, NuverialButtonComponent, NuverialIconCompo
   templateUrl: './confirmation.component.html',
 })
 export class ConfirmationComponent {
-  public loadApplicationId$ = this._route.snapshot.paramMap.get('externalTransactionId');
+  public loadApplicationId$ = this._formTransactionService.transaction$.pipe(
```

Comment:
Application is on old term that we want to make consistent, can you rename
```suggestion
  public transactionExternalId$ = this._formTransactionService.transaction$.pipe(
```

Reply:
I renamed it to `externalTransactionId` to keep it consistent with other parts of the app